theia-extensions/viewer-prototype/package.json

```diff
@@ -21,6 +21,7 @@
     ],
     "dependencies": {
         "@theia/core": "1.45.1",
+        "@theia/messages": "1.45.1",
```

Comment:
optional: move it down according to alphabetical order?

Reply:
Done